docs/tldr/pages/linux/tmt.md

```diff
@@ -0,0 +1,36 @@
+# tmt
+
+> Test Management Tool for creating, running and debugging tests.
+> More information: <https://tmt.readthedocs.io>.
+
+- List available tests, plans, or stories:
+
+`tmt {test|plan|story} ls [<pattern>]`
+
+- Run all test steps for each plan:
+
+`tmt run`
+
+- Run selected plans and tests:
+
+`tmt run plan -n <plan_name> test -n <test_name>`
+
+- Show what tests a specific plan would run:
+
+`tmt run -vv discover plan -n <plan_name>`
+
+- Show test results from the last run in a web browser:
+
+`tmt run -l report --how html --open`
+
+- Add context to a test run:
+
+`tmt -c foo=bar -c baz=qux,quux run [...]`
```

Comment:
I would put `<run|...>` to indicate that you can use this for other things, e.g. `show` would give you the expanded format after `adjust` is applied.

Reply:
Good point.